packages/mobile/package.json

```diff
@@ -29,8 +29,8 @@
     "test:e2e:ios": "./scripts/run_e2e.sh -p ios",
     "test:verify-locales": "./scripts/verify_locales.sh",
     "pre-deploy": "./scripts/pre-deploy.sh",
-    "deploy:update-disclaimer": "yarn licenses generate-disclaimer > android/app/src/main/assets/custom/LicenseDisclaimer.txt",
-    "postinstall": "./scripts/sync_branding.sh"
+    "deploy:update-disclaimer": "yarn licenses generate-disclaimer > src/account/LicenseDisclaimer.txt && ./scripts/copy_license_to_android_assets.sh",
+    "postinstall": "./scripts/sync_branding.sh && ./scripts/copy_license_to_android_assets.sh"
```

Comment:
Instead of postinstall can we do this as part of pre-deploy? That's when the license file gets updated anyway

Reply:
We have to keep the postinstall so that people cloning the repo get the file to the right place.
The alternative was to write the file to both locations during generation, but I favoured something closer to a single source of truth.

Reply:
I see you're reasoning. Part of me still hopes we can eventually stop using postinstalls entirely and turn them off in npm/yarn for the big security benefit. But maybe that's wishful thinking.